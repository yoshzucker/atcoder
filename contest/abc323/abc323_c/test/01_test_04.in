90 90
1300 2000 1000 900 2300 700 1300 2200 900 1500 2000 1100 2100 1900 1000 1600 900 1500 1900 700 1700 2100 1700 1600 1900 1600 2000 1400 1800 500 700 2400 1900 800 1100 900 1600 1400 1900 2000 2200 600 2300 1300 2100 2400 800 2400 1900 500 1000 1600 2400 1600 1300 1900 1800 2500 1900 2300 1200 2200 800 2100 1100 1700 500 500 2400 1300 1700 2200 1100 2400 2100 1800 2300 1200 2400 2300 1200 500 1800 1200 2100 1600 800 1900 800 2300
xoooooxoxxoxoxxxxoxoxoxxxxoxxoxooooxxoxxoooxxoxoxxxxoxxxoxxxxoooxxoooxxxxxxooooxxoxoxoxoxx
ooxooxxoxxxxxooxxxoxxoxoxxxoxxoxxoxxxxoxoxoxxoxxxxxxxoxxooxxoxoxoxxoxxooxooxxooxooxxooooox
xxxooxxoooxoooooxxoxooooxxoooooxxxoxoxooooooxooooxoxxoxxoooooxoxxoxooxoxooooxooxoxooxxooox
oxoxoxxoxxxoxxoxoxoooxxxoxxoooxxooooxxxxxoxooxoxoxxxooxoooooxxoxoooxooooxxxoooooooxoxxxoox
oxoooooooxooooxooxxooxoxoxoxxooooxxxxxoxooxoxxoooxxxoxooooxoxooxoxooxoxoxxxxoxxxxoxoxooxoo
oxxoxooooxxxoxxooxoxoxoxxoxxxoxxooxoxooooooxxoxxxoxoxxooxxxoxxxoxxxoxooxoxooooooxoooxoxxxx
xxooxooxoooooooxxxoooxooxxxoxoxxxxxoxxoxoxooxoxooxxoxoxxooxoxxxxxxxxxxxooxxxxxoxoooxooxooo
xoooooxooooxxxxxxxooooxooxxxoxoxxooooxxoxoxoxxxoxxxxoxxoxooxoooxxoxxoooxoxoooxooooxoxoxxxx
xxxooxxoooxxoxxxxxoxxxoooooxoxxxoooxooxoxoxxoxxoxoxoxxoxxoxoooooxooooooxxxoxooxxxooooxxoxo
xoxoooxoxooxxxxxxxoooxxxoxooxooooxooxooxoxxooooxoxoxooxxxooooxoxoooxoxxxxxxoxxxxxxooxxoxxo
oooxxooxxoxxxooxoxxxxxoxooxxxoxxoxoxooxoxxooooxxxxxoxxxxxxxoxooooxxoooxoxoxxoooxxxxxxxoxxo
ooxxoxooxoxxoooooxxxxxooooxxooxxoxooxoooxxooxxoxxooxxxoooxooxxxoooxxooxoooxxxxxoxooxoxoxxo
xoooxxoooxooxxoxoxooooxoxxxoxooxoooxxooooxxoxxxoooxooxxxoxooxoooxxxxxxxoxxxooxxxooooxxxxox
xoxxxoxxxxxxooxxxxooxoxoxxxxooxxxxxxxxooxxoooxxxooxooxxxxxxooxxxxxxxxxxoxxoooxoxoooxxxoxxx
oooxxooooooxxoxoxoxxoxxxoxoxooxooxxoxooxxooooxxxoxoxxxoxxxxxoxxoxoxxxxxxxxxoooxooxxxooxoxo
xxxxxxoxooxoxooooooxxxxxxxoxxxooooxoxxooxxxoxoooooxooxoxoxxoxxxoxoxoxxxxxxxxoxxxxoxooxoxox
xxxoxxooxoooxooxoooooxxooxoxxoooxxxoxxxxxxooxxoxoooooxxoxoxxxxxooxooxooxxxxooxoxoxxoxoooxo
xoooxooxxxxoxoxxoxoxooxxxoxooxoxoxxxxxxxxxoooxxxoxxxoxxoooxxxxoxxoxoxxooxxoxoxoxxooxoooxox
oooooooxoxxooxoxoxxoooxooooxoxoxoooxxxxxxoxoxoxxxxooxxxoxooxooxxoxxxxxoxxoxooxooxoxooooxxo
xxxxxooxxoooxxoxxxxxxxoooxoxoxxxoxooxooxxxxxoxoxoxooxoxxxxxoxxoxxxxxxxxooooxooxxoxoxxxxxoo
oxoxooooooxooooxxoxxooxooxxooxxxxooxxxoxoxxxoxxoooooooooooxxooxooxxxooxoxoxoxooxxooxxooxoo
ooxooxxxoxoooxooxxxoooooxxooxoxoxoooxooxxxxoooxxoooxxoooxxoooxoxoxoxxxxxooxxxxoxoxxxxxooxx
xxxoxxxxoxooxoooxoxoxxxxxxxxooxxxooxooxxooooooooooxxoooooooxooxoooooxxxoooxooxoooooxxooxoo
ooxxxoxooxoxxoxoooooooxooxxoxxoxxooxxoxxxxxoxxxxooxoooooxxoxooxoooxxxxooxoooooxxxxxxoooxoo
xoooxooooooxxxoxoooooxoxxxooxxxoxoxoxxxoooooxxoooxoxoooxoxxxxxoxooxoxxooxxoxoooooxxoxoxooo
xxxoxooooxxxxoooxxooxoxoxxoxxoxooxxoooxxxooooooxxoxoxxxxooxooxxoxxoxxxoxxoxooxxooooxxoooxo
xxxoxoxxxoxxoxooxxoxxxxooxxoxoxoxoxooooxooxxoxxxxoooxooxxxxxxooooxoxoxoxxxxooxxooooxoxoxox
oooxooxoxxoooxxxxooxxxxxxxoxoooxxoooooxxxxoxxoxooxxxoooxooxoxxxoxxooxoxxooooxoxxooxxxooxxx
oxoxxxxxxoxoxoooxoxxxxooxooxxooooxoooxooxooxxxoxooxooxxxooooxooxxoxoxoxxooxxooooooxoxxxxox
xoxxooxoooxooxxxxooxoxxooxoxoxxooxoooxxooxoxoxxooxooooxoxoxoxoxoxxooooxoxxxoxxoxoxxxoxoooo
xoxoxxoooxxoooxoxoxxooooxoooxoxxxooxooxxoxoxxoxxoxxoooxoooxxxxxxxxoooxoxoxxxxoxxoxooxoxoxx
xxoxxoxxooooxxxooxxoooxxooxooxooxxxooxoooxxxxxoooooxxoooooxoxxxxxxoxxxxxoxxooxxxooxxxooooo
xxxxooxxxxxooxoxoxoooooooxxxxoooxxoxxxxoooxxxoooxxooxoooxooxoooooxoxooxoxoxxxoooooxooxxoxo
xoooooxooooxxoooooxoxoxxxxxxooxxoxxooxooxxoxoxoxooxoooooxxxxxoxxxxxoxxxxxooooxxxxxxoxoxxoo
ooooxxxoooxxxooxoxooxooooooxoxoxoooxooooxxooxoxoxoxxxxooxooooxxooxoxoxxoxxxoxxoxxxxxoxxxoo
xxxoxxxoxoooxxooxxxxooxoooooxoxxooooxoxooxxxoxxxoxxxoooxoooxxxxooooxoxxxoxxxooxxxxxooxoooo
ooooxxxoxxooxoooooxxooooxooxxoxxoxoxxoxoxxooxxoxxxoxoxxoxxoxoxxxoxooxoxooooxooooxxooxoxooo
ooxoxooxoxoxoxxoooxoxxoooooxxxxooooxoxoxoxxoooxooooooooooooxoxoxoxxoooxxxoooxxoooxxoxxoxox
xxoooxxooxxxxooxxxooxooxooooxoxxooxxxxooxoxooxxoxxxxxoxxxxooxoxooxxoxxxxxxxxxooxxxooooxooo
oxooxxxxxxxooooxooxxoooxxxooxooxoxxoooxxoooxoxxooooooxxooooxxxxoxxoooooxoxooooxxooxxxoxxxo
oxxxxoxooxxoxoooxxooooxoxoxoxxxooxxxxooooxxoxooooooooxxooooxxxxooxxoooxoxxxooxoxoxxxoooxox
ooxoxxoxxxooxooxoxxoxoxoooxxxxxxoxoxxxooooxxooooxoxooxxxxoxooxoxxoxoooxxoxoooooxoxoxooxxoo
ooxoooxxoxoxxxxxoxoooxxooooxoxooxxooxoooxxoxxoooxoxooxxoxxxooxoxooxxoxxxxoxooooxooxxxooooo
oxxxoxxoooxoxxxxxooxxxxooxooxxooxxoooooxooooooxxooxoooooxoxoxoxoxooxxxxxxxoxoxxoxxxoooooxx
oxoooxoxxoxxoooxoxooooxxxxxooxoxoxooxxoxoxxooxxxxxxxxoooooxoxoooxoxoooooxxooxxooxoxxoxoxxx
ooxxoxoxooxoxooxooooxoooxxxxoxooxoooxooooxooxxoxxxoxoxooxoxoooxxxoooxxxxooxxxooxoxxxxoooxo
xoooxoooooooxooxxxxooooxxoxxxooxxooooxxxxoxxxxooooxoooxxxoooxxxxxxoxxxoxxoxoooxoxxxoxoxxox
xxxooxoooxooxxoxooooxooxxooxxoooooxxxoxxxooxooxooxxxxoxoxxxxoxxoxxxooooooxooxoxxoxoooooxox
oooxxxoxxoxoxoxxxxoxooxxoooxxooxxxxxxoxxxooxxxoooxooxoxoxxxooxxxoxoooooxxooxxooooxxxooooxx
xxoooxxooxxxoxooxoxoxoxooxxoxoxxxoxxoxxoxooxxoooxoxxoxooxxxoxooxooooxxooxoxoooxoxooxoxoxoo
xxoooxxoxooxoxxxxxoooooxxxxxxxxxooooxxoooxxooxoxoxxxoxoxooxxxxoxoxoooxxoxxooxooxxoxxxxxxxo
xxxoxxxoxoxxxxoxooxxxoxxooooooxxooxoxxxoxxooooxooxoxoooxxxxxooxooxxooxxooxxoxxxoxxxoooooxx
oxxxxxxoxoooooxoooxoooxoxoxoooooxoxxxooxxoooooxxxxooxoooooxooxxoooxxoxxxxoxoxxoxxxoooooooo
xxxoxoxoooxooxooxxooxoxoooxxoxoxxooooxxxoooxxxooxxoxoxxoooooxxxooxoxoxoxooxxoxoxooxoxoxoox
xxxxxoxoooxoooooxxxoxxoxooxxoxooooxooooxxooxoxooxxxxxoxoxoxoxooxoxxoxxxooxooxxxoxxxooxxxoo
xxoxooxoxxxoxxxxoxxoooxoxxoxxoooxoxoxooxoooxxxooooxxooooxxoooxoxooxoxxxxxxxooooxoxoooxxoox
xxooxoxxoxxoooxoooxoxxxooxxxoxooxxoooxoxxoooxooxxoxoooxoxxoxxxoxxooxooxxxooxooooxoxxoxxoxx
xxxoxxooooooxooxxxxxxxxoxxxxxoxxxxxxxxoxooooxooxoxxxoxxxxoxoxooxoxxooxxooxxoxoooxooxooxoox
xoxxoooxxoxooooxxoxoxxooxxxxoxxxxxxxoxoxooxoxxxxoxooxxxxoxxoxoooooxxooxoxxoxooxxxoxooxooxo
xxooxxxooooxxoooooxxxxxxooxxxooxooxxxoxxooxoooxoooooxxooxoxxxxxxoxxoxxxooxoxxoxooxxxxxxoxx
oooxooooxoxxooooooxoxooxxoxxooxxooxxoooooxxxxxooxooxxooxoxxoxoxoooxoxxoxxxoxoxoxxxxoxxooxx
xoxoxxoxooooooxxxxxxoxoxoooxooxoooxxoxoxoxxoooxooxxxoxxxxxxxxxoooxxoxooooooxoxxoxxoxxoxoxx
ooooooxxxxoxooooxxxxoxxxxxxoooxoxoxoxxooxoxooooxoxxoooooxxxxxxxxxxoxoxoxxxxxxxoxxooxxoxoox
ooooxxxooxoxxxxoxooxoxxoxxoxxxoooxxoooxoxxoooooxoxxxooooxooxooooxooxxxxxooooooooxoooxoxoox
xxooooxxxxxxoxoooxxxxoxoooxxooxxxooxooxooxxoxxxxxxxoooxxoxxoxoooxoxxoooxoooooxxoxxooooxxoo
ooooxooxxxoooxxoxxxxxoxoxoxoxxoxoxxooooooooooxxoooxxoxxoxooxooxoxxoxxoxxxoxoxxxoooxxoxxooo
xoxoxxoxoxoooxxooxxxooxxxxxxxoxxoxooxoxoxooooxooxxoooxxoxoooxooxoooooxoooxxoxxoooxooooxooo
xoxoxxooxooxxxxxxoxxoxoxoooxoxxoooooooooxoxoooooxxxxxoxxxxoooxoxxoooxoooxooxxoxooxoxxxoxxx
ooxoxxoooxoxoooooxxxoooxoxoooxxooooxxoxxxxoooxoxxooxxxoooxxxxxoxoooxxxxoxoxxooxxxoxxxxoooo
xoxooxxoxoxxxxoxooxxooxoxooxoxoxoxxoxoxooxxoxoxxxxxooxooooxxxxxooxxoooooxoooxoooooooxxooox
oxoxoxooxoxoxooxxoooxxoxxoxoxoooxxooxxxoxooxxxoooooooxoxxoxxoooooxxooxxoxoxxoxoxxoooxoxooo
ooxooxxxxooxoxxoooxoooxoxxooxxoxoxxxoxoooxoxxxooxxxxooxoxooxoxooxoxxoxooxxooxooxxooxoxxoox
xoooxxoxoxoxxxxooxoxoxoxooooxxxoooooxxooxoxooooxxooxxxxoxooxooxooxxoxoxoxoxxxooooxooxxoxoo
xoxxooxoxoxoxxxxxooxoxoxoxoxxooxxxoooxxxxxxoxxxooxooxoooxoxxooxxooxoooooooxxoxxxoxoxxxxxox
xxxoooxxxxooxxooxxooooxxxoxooxoxoxooxxxxxoooxoxoxoxoxxoxxoxoxooxoooooxooxooooxxxoxooooxxoo
xxxoxoxoxxoooxxxoxxxxooxooxxoooxxooooxoooxoxoooxoxxooxxoxoxxxxooxxoxxxooxxxoxxxxxoooooooxo
oooxxoxooxoxooxoxxxooxoxooooooxoxxxxxooxxooxooxooxoxxoxxoxoxxxooxxxoxoooxxoooooxoxxoxxooox
ooxxoxoooxoxxxoxooxoxooxxoxxoooxooxoxxooxooxoxxooxxxxoxxoxxxxooooxoxoooxxxxoooxoxoxxoxxoxx
xooxoxoxoxoxxooxxxxxoooooxxxxxoxoxoxoxxxooxxxooxxxoxoxxxoxxxxxxoxoxoxxxxooxooooxooxooxoxxo
ooxxxooxxxxxxxooxxxooxoxooxooxoxooxoxoxxxxxoxxxxooxoxoooxxxxoooxxxxxoxxoooooxxoxoxoxoxxxoo
ooooxxooxoxxxxoooooxooxooxooxxxoxoooxooxooxoxxxoxoxoxxoxoxoxoxxxxoxoooxooxxoxooxxoxoxxoooo
xxoooooxooxooxxxxoooxxxxooooxoxoxxooooxoxxxxoooxoxxooooooxxooxoxxxxxoxooooxxxxoooxoxxxxoxx
xoxxxxoxxxxxxoooxoxxxoxxxxoxoxoooxoxoxooxxxxxoxoxooxoxooxoxoxxoxoxooxoxxooxxxooxooxxxoooxx
xoxoxoooooooooooxxxxoooxxxooxxxoxooxooxxooxoxxxxxxxoxxoxxoxxxxxxxxxxooxxoooxxxxooxoxxxxxxo
oxxxooxxooxxxoxooxxxxoxxxoxoxoxoxxooxxoxxxxoxooooooxoxxooooooooxxooooooxooooxoxooxxxooooxo
ooxxoxooooxooxxxxooxooooxooxooxoxxoxooooooxxxoooxoxxoxxxxxxoxxxooxoxoxooooooxxoxooxooxooxo
xxoxoxoooooxxoxoxoooxoxxxxooxxxxooooxxxxxoooxxooxxxxoooxoxoxoooxoxxoxxxxooooxoxooxxxoxxxxo
xxoooxoxxoxoxooxoooxoxoxoxxoooooooxxoxxoxxxooxoxxxoooxoooxxxxoooxxooxxxooxoxooooxxxooxxoox
oooxxxooooxoxooxooxxooxoxoxxxxooxooxxxoxxxoooxooxooxxoxxxxoxoooxxoxxxoooxxoooxoooxoxxxxooo
oxxxooxxoooxoxxxoxxxoxoxxoxxxxoooxxooxxxxxoxoooxxxxxoxxxxooxxooooxoxoxxoooooxooxooxxoxxoxx
