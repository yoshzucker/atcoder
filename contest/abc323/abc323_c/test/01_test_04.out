13
14
0
8
7
10
10
8
7
9
13
6
9
18
10
15
10
13
8
16
3
10
4
7
5
9
13
10
8
3
9
11
6
10
5
9
6
2
11
6
5
6
6
6
7
6
12
7
10
8
12
10
2
7
9
9
10
13
11
14
7
7
13
2
7
6
4
7
9
5
6
5
6
11
7
9
6
9
13
13
7
7
13
13
5
2
10
7
7
11
